crates/autopilot/src/run_loop.rs

```diff
@@ -146,7 +148,19 @@ impl RunLoop {
                     }
                 })
                 .collect(),
-            prices: auction.prices.clone(),
+            tokens: auction
+                .prices
+                .iter()
+                // TODO Fill in the missing fields (Default::default())
```

Comment:
As per my comment ☝️, I think that the token fields that are set to `Default::default` here should not be part of the `auction` JSON.